src/Sylius/Bundle/UserBundle/Resources/config/services.xml

```diff
@@ -28,12 +28,16 @@
         <defaults public="true" />
 
         <service id="Sylius\Bundle\UserBundle\Console\Command\DemoteUserCommand">
+            <argument type="service" id="doctrine" />
+            <argument>%sylius.user.users%</argument>
             <tag name="console.command" />
         </service>
 
         <service id="Sylius\Bundle\UserBundle\Command\DemoteUserCommand" alias="Sylius\Bundle\UserBundle\Console\Command\DemoteUserCommand" />
```

Comment:
Alias can be removed since class aliases are removed.

P.S. Alias can be made deprecated at 1.13/1.14

Reply:
Same for other commands

Reply:
done